src/main/java/com/mg105/user_interface/MapDrawer.java

```diff
@@ -49,7 +49,7 @@ public MapDrawer(@NotNull RoomInterpreter interpreter) {
         tiles.put(TileType.WALL, new Image(Objects.requireNonNull(RoomUpdater.class.getResourceAsStream("/tiles/wall.png"))));
         tiles.put(TileType.EXIT, new Image(Objects.requireNonNull(RoomUpdater.class.getResourceAsStream("/tiles/exit.png"))));
         tiles.put(TileType.CHEST, new Image(Objects.requireNonNull(RoomUpdater.class.getResourceAsStream("/tiles/chest.png"))));
-        tiles.put(TileType.PLAYER, new Image(Objects.requireNonNull(RoomUpdater.class.getResourceAsStream("/tiles/player.png"))));
+        tiles.put(TileType.PLAYER, new Image(Objects.requireNonNull(RoomUpdater.class.getResourceAsStream("/sprites/A.png"))));
```

Comment:
Changed default character sprite. The old png file might be useless now.

Reply:
So delete it?

Reply:
ok